backend/packages/Upgrade/src/api/services/AnalyticsService.ts

```diff
@@ -320,6 +327,7 @@ export class AnalyticsService {
         // write in the file
         const csv = new ObjectsToCsv(csvRows);
         try {
+          experiment.type === EXPERIMENT_TYPE.FACTORIAL? csv.delimiter = ',' : false;
```

Comment:
This ternary is pretty strange, that `false` is meaningless. We're just saying this, right?
```
if (experiment.type === EXPERIMENT_TYPE.FACTORIAL) {
  csv.delimiter = ',';
}
```

Reply:
resolved